Age-ify.js

```diff
@@ -0,0 +1,5 @@
+console.log("Homework-1");
+let yearOfBirth = 1990;
+let yearFuture = 2045;
+let age =  yearFuture-yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
```

Comment:
This could look a bit cleaner with a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). I see you've used one at another of the exercises.